edgelet/Cargo.lock

```diff
@@ -87,7 +87,7 @@ checksum = "cdb031dd78e28731d87d56cc8ffef4a8f36ca26c38fe2de700543e627f8a464a"
 [[package]]
 name = "aziot-cert-client-async"
 version = "0.1.0"
-source = "git+https://github.com/Azure/iot-identity-service?branch=main#f6294bdd027284a27a03d83b4c5dc736d98e0a32"
+source = "git+https://github.com/Azure/iot-identity-service?branch=main#b59977078be65a4cc5d981cc036ced13f69b5f18"
```

Comment:
@arsing, let me know if I missed anything here.

Reply:
Yes, it's correct.